static-assets/scripts/resources/locale-es.json

```diff
@@ -37,6 +37,7 @@
       "ACCOUNT": "Cuenta",
       "SITE_ERROR" : "es un sitio inválido",
       "SITES_PER_PAGE": "Sitios por página",
+      "NO_SITES_MESSAGE":"No eres miembro de ningún sitio. Por favor, póngase en contacto con su administrador para solicitar el acceso.",
```

Comment:
This is mixing two types of language (formal/friendly) - tuteo with ustedeo.

@sumerjabri which tone do we want to take casual/friendly or more formal/respectful?

Reply:
What's the tone of the rest of it?

If unclear, use formal.

Reply:
Done